crates/league-toolkit/Cargo.toml

```diff
@@ -3,6 +3,18 @@ name = "league-toolkit"
 version = "0.1.0"
 edition = "2021"
 
+[features]
+default = ["zstd"]
+
+zstd = ["dep:zstd"]
+ruzstd = ["dep:ruzstd"]
```

Comment:
Why do we need this?

Reply:
Not all environments (i.e wasm) can handle linking to something like zstd. Theoretically, the zstd crate is supposed to work with wasm, but it seems to be really fucked. That, combined with the fact that we also need to use flate2's rust backend in wasm anyway, makes it worth offering a rust-only setup for league-toolkit.

Reply:
Makes sense 👍🏼